src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -71,7 +71,37 @@
  *     value=&quot;^log(ger)?|[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   final static int logMYCONSTANT = 10; // OK
+ *   final static int loggerMYCONSTANT = 50 // OK
+ *   final static int MYCONSTANT = 100; // OK
+ *   final static int myConstant = 1000; // violation, name 'myConstant' must match
+ *                                       // pattern '^log(ger)?|[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
```

Comment:
you found bug in config :) , please fix it.

expected behavior should be:
```
    final static int log = 10; // OK
    final static int logger = 50 // OK
    final static int logMYCONSTANT = 10; // vioaltion
    final static int loggerMYCONSTANT = 50 // violation
    final static int MYCONSTANT = 100; // OK
    final static int myConstant = 1000;
```

Reply:
Done.